src/main/scripts/src/app/view/view-controls/share-dialog/share-dialog.component.ts

```diff
@@ -0,0 +1,126 @@
+/*
+ * -----------------------------------------------------------------------\
+ * Lumeer
+ *
+ * Copyright (C) since 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -----------------------------------------------------------------------/
+ */
+
+import {Component, EventEmitter, OnInit, Output} from '@angular/core';
+import {NotificationsService} from 'angular2-notifications/dist';
+import {UserService} from '../../../core/rest/user.service';
+import {User} from '../../../core/dto/user';
+
+const BACKSPACE_KEY = 8;
```

Comment:
We have these constants already defined.
https://github.com/Lumeer/web-ui/blob/deb906d74a08b85a56066350869e448b018901b9/src/main/scripts/src/app/shared/key-code.ts